apexlog/apexlog.go

```diff
@@ -0,0 +1,66 @@
+package apexlog
+
+import (
+	"fmt"
+
+	"github.com/airbrake/gobrake/v5"
+
+	"github.com/apex/log"
+)
+
+// severity identifies the sort of log: info, warn etc.
+type severity int
+
+// These constants identify the log levels in order of increasing severity.
+// A message written to a high-severity log file is also written to each
+// lower-severity log file.
+const (
+	DebugLog severity = iota
+	InfoLog
+	WarnLog
+	ErrorLog
+	FatalLog
+)
+
+var severityName = []string{
+	DebugLog: "debug",
+	InfoLog:  "info",
+	WarnLog:  "warn",
+	ErrorLog: "error",
+	FatalLog: "fatal",
```

Comment:
When I send `ctx.Fatal("oops")`, it exits the program but never reports anything to Airbrake.

Reply:
The apex/log package exits the application internally denying us to log it in airbrake

Reply:
If we cannot support this, we shouldn't be exposing this severity in our integration because it doesn't do anything useful. In doing so, we should probably document this behavior. If I set up `FataLog` severity, nothing at all would be sent to Airbrake, which is misleading (I would think that only fatal logs would be sent).

Reply:
Ok, I will remove that.